### ISSUE TITLE
No single pointer is provided to drag the "Copilot github Chat Separator" to the required position: A11y_Visual Studio Code Client_Copilot_Copilot Chat_Dragging movements.

### ISSUE THREAD

**Issue by keerthiduvvuri** [microsoft/vscode#223853]
### GIT Hub Tags:
#A11yWCAG2.2; #A11yTCS; #A11ySev3; #Visual Studio Code Client; #BM_Visual Studio Code Client_Win32_JULY2024; #DesktopApp; #WCAG2.5.7; #FTP; #A11yMAS; #Dragging Movements;
### Environment and OS details:
Application Name: Visual Studio Code Client
OS: Windows 11 version 23H2 OS built: 22631.3880.
### Reproduction Steps:
1. Open Visual studio insider
2. In the Visual Studio Code Marketplace, go to the GitHub Copilot extension page and click Install.
3. After installing the extension go to the github copilot chat option available in the left side panel and press enter.
4. Observe that No single pointer is provided to drag the "Copilot github Chat Separator" to the required position.
### Actual behavior:
No single pointer is provided to drag the "Copilot github Chat Separator" to the required position.
### Expected behavior:
There should be single pointer is provided to drag the "Copilot github Chat Separator" to the required position.
### User Impact:
People with motor impairments who cannot operate content because the only way to actuate a function is by dragging a target element from its initial position to some other position will face difficulty in accessing the application.
### Attachments:
[No single pointer is provided win32.webm](https://github.com/user-attachments/assets/6a38c8b0-153a-4977-a893-0bba189a610c)

**Comment by Samhitha66** [comment]
#Rev:samhithaks;

**Comment by roblourens** [comment]
Same as https://github.com/microsoft/vscode/issues/221558?

**Comment by keerthiduvvuri** [comment]
@roblourens this bug is logged for win32. The bug https://github.com/microsoft/vscode/issues/221558 which you mentioned is related to MAC platform.

**Comment by Govardhani-T** [comment]
@roblourens This bug is logged for win32 platform. The bug which you mentioned as duplicate is related to MAC platform. Could you please resolve this issue and assign back to us.

**Comment by Govardhani-T** [comment]
As per the above comment assigning this bug to PO and reopening the issue.
#Reactivated;
